scriptworker/version.py

```diff
@@ -52,7 +52,7 @@ def get_version_string(version):
 
 # 1}}}
 # Semantic versioning 2.0.0  http://semver.org/
-__version__ = (18, 1, 0)
+__version__ = (19, 0, 0)
```

Comment:
This might be 19.1.0 or 20.0.0 now. I think the only potentially backwards-incompatible change is the new dependency.